src/isomorphic/classic/class/ReactClass.js

```diff
@@ -9,6 +9,8 @@
  * @providesModule ReactClass
  */
 
+/* global hasOwnProperty:true */
```

Comment:
Can you please remove them and verify `npm run lint` passes? I tested locally with the existing comments on `master` and it seems to lint fine. I don't think our usage of `hasOwnProperty` requires this to be defined as global with eslint.